vars.tf

```
@@ -376,3 +384,21 @@ variable "db_backup_s3_bucket" {
 variable "db_backup_service_mode" {
   default = "backup"
 }
+
+variable "rclone_arguments" {
+  type        = string
+  default     = ""
+  description = "(optional) e.g., --combined -"
```

Is this variable necessary? If not, please remove it. If so, I'd like to keep to a uniform ordering of attributes.
```suggestion
  description = "(optional) e.g., --combined -"
  type        = string
  default     = ""
```

---

It is optional.  It may be useful during debug to diagnose problems using `rclone`.

---

OK, you can leave it. I'm inclined to wait until it's needed, but it's not a big deal.